web/styles/left_sidebar.css

```diff
@@ -1365,29 +1366,21 @@ li.topic-list-item {
     }
 
     #hide-more-direct-messages {
-        display: block;
+        grid-template-columns: inherit;
+        grid-column: 1 / -1;
+        line-height: var(--line-height-sidebar-row);
         text-decoration: none;
         color: inherit;
         /* 12px at 14px/1em */
-        font-size: 0.8571em;
 
-        & span {
-            display: block;
-            /* TODO: Rewrite the show-more and show-less
-               buttons as grids alongside the left sidebar
-               header rewrites. The 12px left padding here
-               aligns the "back to channels" text with the
-               DIRECT MESSAGES heading above, which itself
-               is offset -3 px to the left, but reserves
-               15px for the arrow marker. 15px - 3px = 12px. */
-            padding: 2px 0 2px 12px;
+        .hide-more-direct-messages-text {
+            font-size: 0.8571em;
+            grid-column: row-content;
```

Comment:
`grid-area` would be a more robust choice than `grid-column`, just in case rewrites to the area include rows in the future.

Reply:
Fixed [here](https://github.com/zulip/zulip/compare/4b42779cde4049045fe520c4cbf0ca2c22db68a9..fb5bafdc8183d722874f5d15e3acd49aadc7ac48).